src/terminal/kitty.rs

```diff
@@ -289,3 +285,12 @@ struct KittyDimension {
     width: u32,
     height: u32,
 }
+
+impl KittyDimension {
+    /// Tests whether or not this dimension is greater than or equal to the specified dimension.
+    fn ge(&self, other: (u32, u32)) -> bool {
+        let (width, height) = other;
+
+        self.width >= width && self.height >= height
+    }
```

Comment:
Honestly I don't like this: It compares unrelated types and doesn't use Rust's builtin order traits.

I'd prefer if you made `KittyDimension` comparable and orderable, and then created a `KittyDimension` from the pair returned by the image create to compare with the actual terminal dimension.